Demo/Environment/Environment.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+struct Environment {
+    
+    // MARK: - Properties
+    
+    static let instanceLocator: String = "<UNSPECIFIED>"
+    
+}
+
+// MARK: - Validation
+
+extension Environment {
+    
+    // MARK: - Internal methods
+    
+    static func validate() {
+        assert(Environment.instanceLocator != "<UNSPECIFIED>", "Please provide above your instance locator in order to run the app. You can find your instance locator Credentials tab of your Dashboard.")
```

Comment:
I it might be better if this behaved the same way as the tests, i.e. it checks for an environment variable and then a file.  Again this is something we can resolve later if you prefer.

Reply:
I was implemented like that initially, but than we discussed that our preference would be to modify a Swift file rather that set an environment variable in the run scheme.

I happy to change that back to environment variables.

Reply:
Leave it for now then.  👍   I'll check out those docs you linked and if we do come up with a better way for this to work then we can address it in the future.